lib/internal/event_target.js

```diff
@@ -276,7 +276,7 @@ class Event {
   }
 
   /**
-   * @type {boolean}
+   * @param {boolean} value
```

Comment:
According to https://stackoverflow.com/a/22276207/6549112 and https://github.com/jsdoc/jsdoc/issues/973, we were using the correct syntax here.
```suggestion
   * @type {boolean}
```